fineract-provider/src/main/java/org/apache/fineract/portfolio/loanaccount/service/LoanWritePlatformServiceJpaRepositoryImpl.java

```diff
@@ -2119,6 +2119,10 @@ public CommandProcessingResult undoWriteOff(Long loanId) {
         }
         LocalDate recalculateFrom = null;
         LoanTransaction writeOffTransaction = loan.findWriteOffTransaction();
+        if (writeOffTransaction == null) {
```

Comment:
You sure about this? The code will not fail if this is null and later on there's a null-check.

Can you pls elaborate?

Reply:
Yep, haven't checked until the end. :-)